src/httpserver.cpp

```diff
@@ -460,8 +461,7 @@ bool StartHTTPServer()
     threadHTTP = std::thread(std::move(task), eventBase, eventHTTP);
 
     for (int i = 0; i < rpcThreads; i++) {
-        std::thread rpc_worker(HTTPWorkQueueRun, workQueue);
-        rpc_worker.detach();
+        threadHTTPWorkers.emplace_back(std::thread(HTTPWorkQueueRun, workQueue));
```

Comment:
No need to create the temporary just to move it into a new std::thread constructor.

Reply:
Right